src/uct/ib/rc/accel/rc_mlx5_iface.c

```diff
@@ -502,7 +519,11 @@ uct_rc_mlx5_iface_init_rx(uct_rc_iface_t *rc_iface,
         return status;
     }
 
-    iface->super.progress = uct_rc_mlx5_iface_progress;
+    if (iface->config.linked_list_srq) {
+        iface->super.progress = uct_rc_mlx5_iface_progress_linked_list;
+    } else {
+        iface->super.progress = uct_rc_mlx5_iface_progress;
```

Comment:
maybe:
uct_rc_mlx5_iface_progress_cyclic_rq
uct_rc_mlx5_iface_progress_ll_rq

and similar naming for post_recv funcs